packages/themes/bmf/src/index.ts

```diff
@@ -1209,6 +1209,9 @@ export const BMF = KoliBri.createTheme('bmf', {
 		:is(a, button):hover {
 			text-decoration-thickness: 0.25em;
 		}
+		:is(a, button):is(:focus, :hover):is(:disabled, [aria-disabled], [disabled]) {
```

Comment:
Warum ist es notwendig, die Styles in alle Themes zu kopieren, wenn schon im Basis-Style enthalten?